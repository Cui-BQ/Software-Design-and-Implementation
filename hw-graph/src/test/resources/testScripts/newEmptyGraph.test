# create a new empty graph object and test its property

# Create a graph
CreateGraph emptyGraph

# List its node
ListNodes emptyGraph

# List child for an non-exist node
ListChildren emptyGraph n1

# Check contains() method for empty graph
Contains emptyGraph n1