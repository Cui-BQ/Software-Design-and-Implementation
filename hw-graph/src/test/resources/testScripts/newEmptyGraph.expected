# create a new empty graph object and test its property

# Create a graph
created graph emptyGraph

# List its node
emptyGraph contains:

# List child for an non-exist node
Exception: java.lang.IllegalArgumentException: Invalid parameter

# Check contains() method for empty graph
emptyGraph contains n1: false